tools/clang/unittests/HLSLExec/LongVectors.h

```diff
@@ -118,9 +127,9 @@ enum ValidationType {
 };
 
 enum BasicOpType {
-  BasicOpType_Binary,
   BasicOpType_Unary,
-  BasicOpType_ScalarBinary,
+  BasicOpType_Binary,
+  BasicOpType_Ternary,
```

Comment:
This comment is actually for line 136:

```c++
enum UnaryOpType { UnaryOpType_Initialize, UnaryOpType_EnumValueCount };
```

Are there going to be more unary ops?

Reply:
Yes, more unary ops are coming.